tools/importer-msgraph-metadata/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/hashicorp/pandora/tools/importer-msgraph-metadata/cmd"
+	"github.com/mitchellh/cli"
+)
+
+const (
+	metadataDirectory        = "../../msgraph-metadata"
+	openApiFilePattern       = "transformed_%s_metadata.xml.yaml"
+	outputDirectory          = "../../data"
+	terraformDefinitionsPath = "../../config/resources"
```

Comment:
we'll probably want to split the AzureAD data sources/resources into their folders in time, fwiw?